src/main/java/com/mg105/Launcher.java

```diff
@@ -0,0 +1,18 @@
+package com.mg105;
+
+/**
+ * Our 'main' class.
+ */
+public class Launcher {
```

Comment:
Cool solution to the previous main not starting the project.

Reply:
If *only* we didn't have to do things like this, alas, woe is me.